models/apartment.rb

```diff
@@ -0,0 +1,94 @@
+require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
+
+apartments = data[:apartments]
+tenants = data[:tenants]
+
+ruby_apartments = []
+ruby_tenants = []
+
+
+class Apartment
```

Comment:
Our class files e.g. `models/apartment.rb` should generally only contain the class definition, and no other code.